monkey/monkey_island/cc/main.py

```diff
@@ -6,6 +6,8 @@
 import time
 import logging
 
+MINIMUM_MONGO_DB_VERSION_REQUIRED = "4.0.0"
```

Comment:
Why 4? I thought we need 3.6

Reply:
I'm not sure what's the absolute minimum, but why not use 4.0.0? The current stable is 4.0.10 ( https://docs.mongodb.com/manual/release-notes/4.0/ ) , and 4.0.0 is *months* old by now.

Reply:
Because Ubuntu 18.04 still provides only 3.6 and people may install that way.
https://launchpad.net/ubuntu/bionic/+source/mongodb

Reply:
✔ That's a good reason